ChangeLog.rst

```diff
@@ -16,6 +16,12 @@ Note worthy changes
 - It is now possible to manage OpenID Connect providers via the Django
   admin. Simply add a `SocialApp` for each OpenID Connect provider.
 
+- The CERN provider has been updated to use OIDC, in order to comply with
+  the planned deprecation of the current Single Sign-On solution. This change
+  introduces a breaking change: egroups are no longer supported, and your
+  application must be modified to use Roles. See the official CERN Auth docs
+  for more information.  
+
```

Comment:
I think it would be good if you could add the documentation wrt to `uid` changes to the list of backwards incompatible changes in the changelog.

Reply:
Will do

Reply:
Does this look better now? Do you think it needs any rephrasing? In my head is clear what I want to say, but I can't judge how clearly I'm explaining it.